### ISSUE TITLE
Incorrect role given to loader

### ISSUE THREAD

**Issue by hrrcox** [Sage/carbon#6956]
### Description

Loaders have a role of progressbar which fails WCAG 1.3.1 Info and relationships.

### Reproduction

https://carbon.sage.com/?path=/docs/loader--docs

### JIRA ticket numbers (Sage only)

SBS-68239

### Suggested solution

Role should be removed, instead use visually hidden text of 'loading' for screen reader user.

### Carbon version

142.9.1

### Relevant browsers

Safari

### Relevant OSs

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
FE-6820
